static/js/utils/utils.js

```diff
@@ -198,10 +198,24 @@ function getTimeUntil(time) {
     var now = Date.now()
     var tdiff = time - now
 
+    var negTimeUntil = false
```

Comment:
The problem is that we should round up instead of rounding down when the time difference is smaller than zero. So I would suggest to use a simpler approach:
```
function getTimeUntil(time) {
    let now = Date.now()
    let tdiff = time - now
    let roundFunction = tdiff < 0 ? Math.ceil : Math.floor

    let sec = roundFunction((tdiff / 1000) % 60)
    let min = roundFunction((tdiff / 1000 / 60) % 60)
    let hour = roundFunction((tdiff / (1000 * 60 * 60)) % 24)
    
    return {
        total: tdiff,
        hour: hour,
        min: min,
        sec: sec,
        now: now,
        time: time
    }
}
```